src/rules/no-unused-modules.js

```diff
@@ -0,0 +1,720 @@
+/**
+ * @fileOverview Ensures that modules contain exports and/or all 
+ * modules are consumed within other modules.
+ * @author René Fermann
+ */
+
+import Exports from '../ExportMap'
+import { makeOptionsSchema } from 'eslint-module-utils/moduleVisitor'
+import resolve from 'eslint-module-utils/resolve'
+import docsUrl from '../docsUrl'
+
+// eslint/lib/util/glob-util has been moved to eslint/lib/util/glob-utils with version 5.3
+let listFilesToProcess
+try {
+  listFilesToProcess = require('eslint/lib/util/glob-util').listFilesToProcess
+} catch (err) {
+  listFilesToProcess = require('eslint/lib/util/glob-utils').listFilesToProcess
+}
+
+const EXPORT_DEFAULT_DECLARATION = 'ExportDefaultDeclaration'
+const EXPORT_NAMED_DECLARATION = 'ExportNamedDeclaration'
+const EXPORT_ALL_DECLARATION = 'ExportAllDeclaration'
+const IMPORT_DECLARATION = 'ImportDeclaration' 
+const IMPORT_NAMESPACE_SPECIFIER = 'ImportNamespaceSpecifier'
+const IMPORT_DEFAULT_SPECIFIER = 'ImportDefaultSpecifier' 
+const VARIABLE_DECLARATION = 'VariableDeclaration'
+const FUNCTION_DECLARATION = 'FunctionDeclaration'
+const DEFAULT = 'default'
+const UNDEFINED = 'undefined'
```

Comment:
Let’s get rid of this variable - it makes the code harder to read and also defeats engine optimizations which know how to optimize direct comparisons to strings.

Reply:
Removed with 2922910.